integration-tests/mysql_test.go

```diff
@@ -74,76 +74,95 @@ var _ = Describe("Mysql", func() {
 			)
 		})
 
-		It("should allow setting of database name", func() {
-			broker.Provision("csb-google-mysql", "small", map[string]interface{}{"db_name": "foobar"})
+		It("should allow setting properties do not defined in the plan", func() {
+			broker.Provision("csb-google-mysql", "small", map[string]any{
+				"credentials":           "fake-credentials",
+				"project":               "fake-project",
+				"instance_name":         "fakeinstancename",
+				"db_name":               "fake-db_name",
+				"region":                "asia-northeast1",
+				"authorized_network":    "fake-authorized_network",
+				"authorized_network_id": "fake-authorized_network_id",
+			})
 
-			Expect(mockTerraform.FirstTerraformInvocationVars()).To(HaveKeyWithValue("db_name", "foobar"))
+			Expect(mockTerraform.FirstTerraformInvocationVars()).To(
+				SatisfyAll(
+					HaveKeyWithValue("credentials", "fake-credentials"),
+					HaveKeyWithValue("project", "fake-project"),
+					HaveKeyWithValue("instance_name", "fakeinstancename"),
+					HaveKeyWithValue("db_name", "fake-db_name"),
+					HaveKeyWithValue("region", "asia-northeast1"),
+					HaveKeyWithValue("authorized_network", "fake-authorized_network"),
+					HaveKeyWithValue("authorized_network_id", "fake-authorized_network_id"),
+				),
+			)
 		})
 
 		It("should not allow changing of plan defined properties", func() {
 			_, err := broker.Provision("csb-google-mysql", "small", map[string]interface{}{"cores": 5})
 
 			Expect(err).To(MatchError(ContainSubstring("plan defined properties cannot be changed: cores")))
 		})
+
+		DescribeTable("property constraints",
```

Comment:
I moved this node to the provisioning node. It is within the same operation and seems more coherent, if you think we should create it as an independent node, let me know.
Current node scheme:
`MySQL => provisioning => property constraints`